R/write.report.R

```diff
@@ -117,20 +117,11 @@ prepareData <- function(x, model = NULL, scenario = NULL, unit = NULL, skipempty
   }
   for (i in grep(sep, names(x), fixed = TRUE, useBytes = TRUE)) x <- colsplit(x, i, sep = sep)
 
-  unitsplit <- function(x, col) {
-    w <- grepl("\\(.*\\)", x[[col]])
-    x[[col]][!w] <- paste0(x[[col]][!w], " (N/A)")
-    key <- "(.*?) \\((([^\\|]*))\\)($|\\.)"
```

Comment:
`($|\\.)`
why the test for a literal dot instead of end-of-line? My new regex doesn't have this. The old regex would match:
`some thing (unit).|other thing`
and would then take `some thing` as the var name, and `unit` as the unit, and discard `other thing`. Is that on purpose?

Reply:
It is not a dot, it is `\\.`, so it wouldn't match spaces or tabs after the unit bracket, only literal dots.

Reply:
I tested default REMIND and MAgPIE post-processing and looked through the NAVIGATE and AR6 templates for reporting and found no variable names which needed the literal dot after the unit. We decided to use the new regex without the dot option since it can lead to hard-to-detect bugs and seems unnecessary.